projects/optic/src/commands/diff/diff.ts

```diff
@@ -177,11 +181,14 @@ const runDiff = async (
       logger.info(log);
     }
 
-    if (!config.isInCi) {
-      logger.info('');
-      logger.info(
-        `Configure check standards in optic cloud or your local optic.dev.yml file.`
-      );
+    logger.info('')
+
+    if (!hasOpticUrl) {
+      logger.info(chalk.blue.bold(`See the full history of this API by running "optic add ${path.parse(baseFile.sourcemap.rootFilePath).base}"`))
+    }
+
+    if (!config.isInCi && specResults.results.length) {
```

Comment:
this is a little annoying -- you may see it after setting up CI, but hard to know if you have the GH action set up from here w/o running the API. I think we can figure out what to do as we hear from users.

Reply:
This isn't going to do what you expect - this will always show up if your running this locally, regardless whether you have set something up in CI or not. (`config.isInCi` refers to wheter the current run is in CI (i.e. `process.env.CI === true`)

I suggest something different - how about:
- at the end of a diff run ask if you want to set up optic in CI and gives you three options
  - yes (runs optic ci setup) - e.g. running something like https://github.com/opticdev/optic/blob/main/projects/optic/src/commands/ci/setup.ts#L83-L87
  - maybe later (asks again in 1 week)
  - no (stops asking)

We can store this in `~/.config/optic/config.json` https://github.com/opticdev/optic/blob/main/projects/optic/src/config.ts#L190-L213

Reply:
That feels a bit too in your face -- going right into a prompt at the end of a diff. Let's think a bit more on this today

Reply:
Alternatively, we can log comments and do the same idea:
- will keep asking you every 1 week until you either:
  - run `optic ci setup`
  - run `optic mute ...` or something like that